lib/system.js

```diff
@@ -0,0 +1,23 @@
+import Actor from './actor';
+import _debug from 'debug';
+import ProcessPool from './process-pool';
+/* eslint-disable */
+import regeneratorRuntime from 'regenerator-runtime';
+/* eslint-enable */
+
+const debug = _debug('ara:system');
+
+export default class System {
+  constructor() {
+    this.processPool = new ProcessPool();
+  }
+
+  async schedule() {
```

Comment:
It's fine. I suspect if we want more functionality, it will be my specifying a specific type of priority queue. But this API most likely wont need to do much.

The only thing we should consider, is the ability to cancel work. But we can do that in a later roud.